api/src/backend/api/v1/urls.py

```diff
@@ -128,11 +130,17 @@
     ),
     # Allauth SAML endpoints for tenants
     path("accounts/", include("allauth.urls")),
+    path(
+        "api/v1/accounts/saml/<organization_slug>/login/",
+        CustomSAMLLoginView.as_view(),
+        name="saml_login",
+    ),
     path(
         "api/v1/accounts/saml/<organization_slug>/acs/finish/",
         TenantFinishACSView.as_view(),
         name="saml_finish_acs",
     ),
```

Comment:
I just noticed. Is this ok? All of these urls are added to the API router so the `api/v1/` prefix should be added by default.

Reply:
In this case it works, but if it’s being added by default, it should be safe to remove and test